package.json

```diff
@@ -57,6 +57,7 @@
     "shared/**/*.{ts,tsx}": "npm run lint:shared"
   },
   "dependencies": {
+    "chartjs-react": "^3.4.0",
```

Comment:
lock lib version

Reply:
this is front end only package

Reply:
Do you realy need this package? You instal 3 chart js packages